crypto/crypto_test.go

```diff
@@ -121,11 +121,18 @@ func TestCrypto_Configure(t *testing.T) {
 		storageType := reflect.TypeOf(client.Storage).String()
 		assert.Equal(t, "*storage.fileSystemBackend", storageType)
 	})
+	t.Run("error - no backend in strict mode is now allowd", func(t *testing.T) {
```

Comment:
Nitpick: maybe 'error - backend must be explicitly set in strict mode' as it's really confusing to read like this 😅